cranelift/native/src/lib.rs

```diff
@@ -105,16 +105,15 @@ pub fn builder_with_options(infer_native_flags: bool) -> Result<isa::Builder, &'
         }
     }
 
-    // `stdsimd` is necessary for std::is_aarch64_feature_detected!().
-    #[cfg(all(target_arch = "aarch64", feature = "stdsimd"))]
```

Comment:
Can the stdsimd feature be removed from `Cargo.toml`?

Reply:
It's not there.